### ISSUE TITLE
[Suggestion] Auto pick up?

### ISSUE THREAD

**Issue by SubjectHawk66** [Tschipp/BuildersBag#3]
I think it's odd that it flawlessly draws items from the bag but doesn't flawlessly put them back in when you break it and pick it up. Was that on purpose?

**Comment by Tschipp** [comment]
Yes, that is on purpose. It's not a /dank/null/ after all. But to me this sounds like a perfect idea for an addon.

**Comment by SubjectHawk66** [comment]
Yes I’d get that, I like this a lot more than the dank null. When I saw the reddit post I freaked, my girlfriend had to calm me down. Then I yelled to her “WE’RE ADDING THIS TO OUR MODPACK”
